README.md

```diff
@@ -24,14 +23,69 @@ If you have downloaded the repo without using the `--recurse-submodules` argumen
 git submodule update --init --recursive
 ```
 
+## MQTT Config File
```

Comment:
As coreMQTT Agent does not expose a config file, it would be better to rename this section.
```suggestion
## MQTT Agent Configurations
```